tasks/tangd-custom-port.yml

```diff
@@ -0,0 +1,53 @@
+---
+# This task tells SELinux for the port that we want the tangd service to use
+- name: Allow the custom port for tangd_port_t in SELinux
+  community.general.seport:
```

Comment:
please use the `selinux` system role - https://github.com/linux-system-roles/selinux
```yaml
- name: Allow the custom port for tangd_port_t in SELinux
  include_role:
    name: linux-system-roles.selinux
  vars:
    selinux_ports:
      - ports: "{{ nbde_server_port }}"
        proto: tcp
        setype: tangd_port_t
        state: present
```